### ISSUE TITLE
AttributeError: 'SwiftStorage' object has no attribute 'connection'

### ISSUE THREAD

**Issue by Austriker** [blacktorn/django-storage-swift#52]
Hi,

I am having issues using django-storage-swift : 

``` sh
Python 3.5.2 (default, Jul  5 2016, 12:43:10) 
[GCC 5.4.0 20160609] on linux
Type "help", "copyright", "credits" or "license" for more information.
(InteractiveConsole)
>>> from django.core.files.storage import default_storage
>>> default_storage.connection
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "/venv/lib/python3.5/site-packages/django/utils/functional.py", line 235, in inner
    return func(self._wrapped, *args)
AttributeError: 'SwiftStorage' object has no attribute 'connection'
>>> 

```

I am running  django 1.10.1 with python 3.5

My settings are : 

``` python

DEFAULT_FILE_STORAGE='swift.storage.SwiftStorage'
STATICFILES_STORAGE ='swift.storage.StaticSwiftStorage'

SWIFT_AUTH_URL='https://auth.cloud.ovh.net/v2.0/'
SWIFT_AUTH_VERSION='2'

SWIFT_TENANT_ID=''
SWIFT_TENANT_NAME=''

SWIFT_CONTAINER_NAME=''

SWIFT_USERNAME=''
SWIFT_KEY=''

SWIFT_EXTRA_OPTIONS = {
    "region_name": "SBG1",
}
```

Thanks for your help.

**Comment by einarf** [comment]
Did you ever figure out what was wrong?

**Comment by Austriker** [comment]
No I didn't.

**Comment by einarf** [comment]
@Austriker Fixed in #64

The correct way to test the backend should now be:
```
python manage.py shell
from django.core.files.storage import default_storage
default_storage.http_conn
```
.. so I guess `connection` was renamed to `http_conn` some time back.
